service/eventer/projects.go

```diff
@@ -1,35 +0,0 @@
-package eventer
```

Comment:
I don't understand why this was all removed

Reply:
I did not see it being used anywhere

Reply:
I can keep it anyway but i'm not sure it's even used

Reply:
@giantswarm/team-biscuit do you know if this is still used?

Reply:
If it compiles then probably no?